src/docker-in-docker/install.sh

```diff
@@ -20,6 +20,7 @@ INSTALL_DOCKER_COMPOSE_SWITCH="${INSTALLDOCKERCOMPOSESWITCH:-"true"}"
 MICROSOFT_GPG_KEYS_URI="https://packages.microsoft.com/keys/microsoft.asc"
 DOCKER_MOBY_ARCHIVE_VERSION_CODENAMES="bookworm buster bullseye bionic focal jammy noble"
 DOCKER_LICENSED_ARCHIVE_VERSION_CODENAMES="bookworm buster bullseye bionic focal hirsute impish jammy noble"
+IP6_TABLES="${IP6TABLES:-""}"
```

Comment:
Can we publicize this Feature option as mentioned in https://github.com/devcontainers/features/issues/1023#issuecomment-2221238973?

Reply:
done